cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -38,6 +38,14 @@ kind: Pod
 metadata:
   name: kube-proxy
   namespace: kube-system
+  # This annotation lowers the possibility that kube-proxy gets evicted when the
+  # node is under memory pressure, and prioritizes it for admission, even if
```

Comment:
This implementation looks like it just considers critical pods for memory pressure, though.